easybuild/easyblocks/g/gurobi.py

```diff
@@ -55,22 +55,25 @@ def __init__(self, *args, **kwargs):
         """Easyblock constructor, define custom class variables specific to Gurobi."""
         super(EB_Gurobi, self).__init__(*args, **kwargs)
 
-        self.license_file = self.cfg['license_file']
+        # make sure license file is available
+        self.orig_license_file = self.cfg['license_file']
+        if self.orig_license_file is None:
+            self.orig_license_file = os.getenv('EB_GUROBI_LICENSE_FILE', None)
 
         if self.cfg['copy_license_file']:
             self.license_file = os.path.join(self.installdir, 'gurobi.lic')
+        else:
+            self.license_file = self.orig_license_file
 
     def install_step(self):
         """Install Gurobi and license file."""
-
-        # make sure license file is available
-        if self.cfg['license_file'] is None or not os.path.exists(self.cfg['license_file']):
-            raise EasyBuildError("No existing license file specified: %s", self.cfg['license_file'])
-
         super(EB_Gurobi, self).install_step()
 
         if self.cfg['copy_license_file']:
-            copy_file(self.cfg['license_file'], self.license_file)
+            if self.orig_license_file is None or not os.path.exists(self.cfg['license_file']):
```

Comment:
when using the environment variable, this will crash on `os.path.exists(self.cfg['license_file'])` because `self.cfg['license_file']` is `None`

Reply:
that should have been os.path.exists(self.orig_license_file), sorry, one sec